pydal/parsers/__init__.py

```diff
@@ -98,7 +99,7 @@ def _default(self, value, field_type):
 
     def get_parser(self, field_type):
         key = REGEX_TYPE.match(field_type).group(0)
-        return self.registered.get(key, self._default)
```

Comment:
If you look at the diff, `self.registered` is now a `defaultdict`, so in case the key is missing is filling the dictionary with the default value.